examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -1926,6 +1923,9 @@ pxScene2d::pxScene2d(bool top, pxScriptView* scriptView)
   // rtPermissions accounts parent scene permissions too
   mPermissions = new rtPermissions(mOrigin.cString());
 #endif
+#ifdef ENABLE_ACCESS_CONTROL_CHECK
+  mCORS = new rtCORS(mOrigin.cString());
+#endif
```

Comment:
It looks like ENABLE_ACCESS_CONTROL_CHECK will no longer be defined.  Should we remove this block of code or is the intention to always setup CORS here (or perhaps some other logic)?

Reply:
Previously ENABLE_ACCESS_CONTROL_CHECK was defined in 3 different CMakeLists.txt and all CORS related code was wrapped in ifdef-s. Now it is used only in this place. If turned off mCORS will be NULL making it clear that CORS is off. By default it is on in examples/pxScene2d/src/CMakeLists.txt